spec/controllers/application_controller/policy_support_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe ApplicationController do
+  describe "#assign_policies" do
+    let(:admin_user) { FactoryGirl.create(:user, :role => "super_administrator") }
+    let(:host)       { FactoryGirl.create(:host) }
+    before do
+      EvmSpecHelper.create_guid_miq_server_zone
+
+      login_as admin_user
+      allow(User).to receive(:current_user).and_return(admin_user)
+      allow(controller).to receive(:assert_privileges)
+      allow(controller).to receive(:find_checked_items).and_return([host.id])
```

Comment:
Maybe it would be better to set the instance variable correctly with something like 
```ruby 
controller.instance_variable_set(:@_params, {:id=> host.id,})
```
than mocking the method this way.

@skateman?